src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -1096,11 +1115,31 @@ function RemoteFunctions(config, remoteWSPort) {
         }
     }
 
+    var _hoverHighlight;
+
+    function onDocumentHover(event) {
+        var element = event.target;
+
+        if (!element) {
+            return;
+        }
+
+        if (!_hoverHighlight) {
+            _hoverHighlight = new Highlight();
+        }
+
+        _hoverHighlight.clear();
+        _hoverHighlight.add(element, false, true);
+    }
+
 
     function createWebSocket() {
         _ws = new WebSocket("ws://localhost:" + remoteWSPort);
         _ws.onopen = function () {
             window.document.addEventListener("click", onDocumentClick);
+            if (!experimental && config.stickyHighlight) {
+                window.document.addEventListener("mouseover", onDocumentHover);
```

Comment:
I think you would need to listen in the capture phase.

Reply:
Also, when updateConfig is called, the event listener should be attached even at that time and disengaged when the config changes again.